samples/MvcApp/packages.config

```diff
@@ -16,17 +16,48 @@
   <package id="Microsoft.AspNet.WebApi.Core" version="5.2.7" targetFramework="net472" />
   <package id="Microsoft.AspNet.WebApi.WebHost" version="5.2.7" targetFramework="net472" />
   <package id="Microsoft.AspNet.WebPages" version="3.2.7" targetFramework="net472" />
+  <package id="Microsoft.AspNetCore.Cryptography.Internal" version="6.0.7" targetFramework="net472" />
+  <package id="Microsoft.AspNetCore.DataProtection" version="6.0.7" targetFramework="net472" />
+  <package id="Microsoft.AspNetCore.DataProtection.Abstractions" version="6.0.7" targetFramework="net472" />
+  <package id="Microsoft.AspNetCore.DataProtection.Extensions" version="6.0.7" targetFramework="net472" />
+  <package id="Microsoft.AspNetCore.Hosting.Abstractions" version="2.1.1" targetFramework="net472" />
+  <package id="Microsoft.AspNetCore.Hosting.Server.Abstractions" version="2.1.1" targetFramework="net472" />
+  <package id="Microsoft.AspNetCore.Http.Abstractions" version="2.1.1" targetFramework="net472" />
+  <package id="Microsoft.AspNetCore.Http.Features" version="2.1.1" targetFramework="net472" />
+  <package id="Microsoft.Bcl.AsyncInterfaces" version="6.0.0" targetFramework="net472" />
   <package id="Microsoft.CodeDom.Providers.DotNetCompilerPlatform" version="3.6.0" targetFramework="net48" />
+  <package id="Microsoft.Extensions.Configuration.Abstractions" version="6.0.0" targetFramework="net472" />
+  <package id="Microsoft.Extensions.DependencyInjection" version="6.0.0" targetFramework="net472" />
+  <package id="Microsoft.Extensions.DependencyInjection.Abstractions" version="6.0.0" targetFramework="net472" />
+  <package id="Microsoft.Extensions.FileProviders.Abstractions" version="6.0.0" targetFramework="net472" />
+  <package id="Microsoft.Extensions.Hosting.Abstractions" version="6.0.0" targetFramework="net472" />
+  <package id="Microsoft.Extensions.Logging.Abstractions" version="6.0.1" targetFramework="net472" />
+  <package id="Microsoft.Extensions.Options" version="6.0.0" targetFramework="net472" />
+  <package id="Microsoft.Extensions.Primitives" version="6.0.0" targetFramework="net472" />
   <package id="Microsoft.jQuery.Unobtrusive.Validation" version="3.2.11" targetFramework="net472" />
   <package id="Microsoft.Owin" version="4.2.1" targetFramework="net472" />
   <package id="Microsoft.Owin.Host.SystemWeb" version="4.2.1" targetFramework="net472" />
   <package id="Microsoft.Owin.Security" version="4.2.1" targetFramework="net472" />
   <package id="Microsoft.Owin.Security.Cookies" version="4.2.1" targetFramework="net472" />
+  <package id="Microsoft.Owin.Security.Interop" version="2.1.2" targetFramework="net472" />
```

Comment:
https://www.nuget.org/packages/Microsoft.Owin.Security.Cookies.Interop/#versions-body-tab

This is flagged as deprecated and unlisted. We'll want to fix that for the latest 2.1 version.

Reply:
Per teams chat, we aren't using the cookies interop, just the base security.interop which isn't deprecated thankfully